stable/prometheus/templates/server-storageclass.yaml

```diff
@@ -0,0 +1,16 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  labels:
+    app: {{ template "prometheus.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: "{{ .Values.server.name }}"
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ .Values.server.persistentVolume.storageClass }}
+provisioner: kubernetes.io/gce-pd
```

Comment:
This needs to be specified in values.  The Kubernetes cluster may not have access to gce-pd, i.e. it is not running in Google Cloud.

Reply:
done